workflow/json-visualaizer/dist/js/main.js

```diff
@@ -0,0 +1,84 @@
+const inputField = document.querySelector(".json__input");
+const jsonOutput = document.querySelector(".json__output");
+const button = document.querySelector(".json__button");
+button.addEventListener("click", () => {
+    const jsonString = inputField.value;
+    if (isJsonString(jsonString)) {
+        const jsonObj = JSON.parse(jsonString);
+        jsonOutput.innerHTML = "";
+        Object.entries(jsonObj).forEach(([key, value]) => {
+            jsonOutput.appendChild(renderStroke(key, value));
+        });
+    } else {
+        jsonOutput.innerHTML = "Not Valid json string";
+    }
+
+});
+
+function isJsonString(str) {
+    try {
+        JSON.parse(str);
+    } catch (e) {
+        return false;
+    }
+    return true;
+}
+
+function renderStroke(key, value) {
+    const objStroke = document.createElement("div");
+    objStroke.className = "obj__stroke" + (typeof (value) === "object" && value != null ? " accordion" : "");
+    objStroke.addEventListener("click", function () {
+        event.stopPropagation();
```

Comment:
I dont see any declarations of _event_ variable. It can cause a lot of troubles, please use an event which is passed to callback as a parameter. [Here is an example](https://developer.mozilla.org/ru/docs/conflicting/Web/API/EventTarget/addEventListener#javascript)